app/sidekiq/publish_translation_job.rb

```diff
@@ -0,0 +1,12 @@
+class PublishTranslationJob
```

Comment:
it doesn't look like this job is correctly following the unique job constraint. I was able to quickly queue up 5 of the same request. I might just be misunderstanding how the unique job constraint works/is displayed in the sidekiq console

<img width="1256" alt="Screenshot 2023-11-07 at 1 40 28 PM" src="https://github.com/CruGlobal/mobile-content-api/assets/47818/84640de2-566b-4007-b066-2cbe0a026861">

Reply:
This has been fixed